tests/xtro-sharpie/iOS-Metal.ignore

```diff
@@ -128,5 +124,112 @@
+!incorrect-protocol-member! MTLLibrary::newIntersectionFunctionWithDescriptor:completionHandler: is REQUIRED and should be abstract
+!incorrect-protocol-member! MTLLibrary::newIntersectionFunctionWithDescriptor:error: is REQUIRED and should be abstract
+!incorrect-protocol-member! MTLLibrary::type is REQUIRED and should be abstract
+!incorrect-protocol-member! MTLBlitCommandEncoder::resolveCounters:inRange:destinationBuffer:destinationOffset: is REQUIRED and should be abstract
+!incorrect-protocol-member! MTLBlitCommandEncoder::sampleCountersInBuffer:atSampleIndex:withBarrier: is REQUIRED and should be abstract
+!incorrect-protocol-member! MTLCommandBuffer::accelerationStructureCommandEncoder is REQUIRED and should be abstract
+!incorrect-protocol-member! MTLCommandBuffer::blitCommandEncoderWithDescriptor: is REQUIRED and should be abstract
+!incorrect-protocol-member! MTLCommandBuffer::computeCommandEncoderWithDescriptor: is REQUIRED and should be abstract
+!incorrect-protocol-member! MTLCommandBuffer::errorOptions is REQUIRED and should be abstract
+!incorrect-protocol-member! MTLCommandBuffer::logs is REQUIRED and should be abstract
+!incorrect-protocol-member! MTLCommandBuffer::resourceStateCommandEncoderWithDescriptor: is REQUIRED and should be abstract
+!incorrect-protocol-member! MTLCommandQueue::commandBufferWithDescriptor: is REQUIRED and should be abstract
+!incorrect-protocol-member! MTLComputePipelineState::functionHandleWithFunction: is REQUIRED and should be abstract
+!incorrect-protocol-member! MTLComputePipelineState::newComputePipelineStateWithAdditionalBinaryFunctions:error: is REQUIRED and should be abstract
+!incorrect-protocol-member! MTLComputePipelineState::newIntersectionFunctionTableWithDescriptor: is REQUIRED and should be abstract
+!incorrect-protocol-member! MTLComputePipelineState::newVisibleFunctionTableWithDescriptor: is REQUIRED and should be abstract
+!incorrect-protocol-member! MTLFunction::options is REQUIRED and should be abstract
+!incorrect-protocol-member! MTLResourceStateCommandEncoder::updateFence: is REQUIRED and should be abstract
+!incorrect-protocol-member! MTLResourceStateCommandEncoder::updateTextureMapping:mode:indirectBuffer:indirectBufferOffset: is REQUIRED and should be abstract
+!incorrect-protocol-member! MTLResourceStateCommandEncoder::updateTextureMapping:mode:region:mipLevel:slice: is REQUIRED and should be abstract
+!incorrect-protocol-member! MTLResourceStateCommandEncoder::updateTextureMappings:mode:regions:mipLevels:slices:numRegions: is REQUIRED and should be abstract
+!incorrect-protocol-member! MTLResourceStateCommandEncoder::waitForFence: is REQUIRED and should be abstract
+!incorrect-protocol-member! MTLRenderCommandEncoder::sampleCountersInBuffer:atSampleIndex:withBarrier: is REQUIRED and should be abstract
+
+# introspection reports them as failing https://github.com/xamarin/maccore/issues/2312
```

Comment:
Metal is a framework where it's very common for selector not to respond :(
You need to try them on device before ignoring them, if they can be called then they should be included